### PR TITLE
Development

### PR DESCRIPTION
**Pull request by darshanjayadev**

### CODE REVIEW

File: bubble_sort/lib/bubble_sort_by.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+def bubble_sort_by(array)
```
Comment (inhaq) [inline bubble_sort/lib/bubble_sort_by.rb:1]: The function is working as it should but it's returning an error when the block is not provided. Kindly fix it so it should work without block as well. You can always use a spaceship operator for comparison as well.

**Review by inhaq — CHANGES_REQUESTED** [review]
Welcome to Ruby world. You did a great job on this project.

Bubble sort:
Kindly follow the comment and fix accordingly. Once fixed, kindly submit for review.

Enumerable:
Good job on this one. Definitely you're improved with practicing and more readings. 👍🏻 for this one. No need to submit for review but I would recommend to add `#my_inject` as well.

File: bubble_sort/lib/bubble_sort_by.rb (hunk 1 above)
Comment (darshanjayadev) [reply bubble_sort/lib/bubble_sort_by.rb:1]: @inhaq Fixed 👍

**Review by inhaq — APPROVED** [review]
### It's 👍🏻
Looks good now. Before merging kindly fix linter errors.
Good luck with the next project.

**Pull request merged**